src/soilbiogeochem/SoilBiogeochemDecompCascadeBGCMod.F90

```diff
@@ -683,7 +729,7 @@ subroutine decomp_rate_constants_bgc(bounds, num_soilc, filter_soilc, &
     real(r8):: Q10                          ! temperature dependence
     real(r8):: froz_q10                     ! separate q10 for frozen soil respiration rates.  default to same as above zero rates
     real(r8):: decomp_depth_efolding        ! (meters) e-folding depth for reduction in decomposition [
-    integer :: c, fc, j, k, l
+    integer :: c, fc, j, k, l, dt
```

Comment:
dt is the time step rather than one of the indices. So I'd define it on it's own line and state what it is.